src/com/cje/RequestPage.java

```diff
@@ -1,17 +1,633 @@
+                        birthHeightTxt.requestFocus();
+                        return;
+                    }
+                    else {
+                        birthHeight = Float.parseFloat(birthHeightTxt.getText());
+                    }
+                }
+                else {
+                    birthHeight = -1;
+                }
+
+                // Check Date of Birth
+                if(ageByDay<0) {
+                    Utils.showMessage("Please input date of birth correctly.");
+                    birthDateTxt.requestFocus();
+                    return;
+                }
+
+                // Check Height value
+                if(heightTxt.getText().length()>0) {
+                   if(!Utils.isActualNumber(heightTxt.getText())) {
+                       Utils.showMessage("Please input number correctly");
+                       heightTxt.requestFocus();
+                       return;
+                   }
+                   else {
+                       height = Float.parseFloat(heightTxt.getText());
+                   }
+                }
+                else {
+                   height = -1;
+                }
+
+                // Check Weight value
+                if(weightTxt.getText().length()>0) {
+                    if(!Utils.isActualNumber(weightTxt.getText())) {
+                        Utils.showMessage("Please input number correctly");
+                        weightTxt.requestFocus();
+                        return;
+                    }
+                    else {
+                        weight = Float.parseFloat(weightTxt.getText());
+                    }
+                }
+                else {
+                    weight = -1;
+                }
+
+                // Check the girth of Head value
+                if(headTxt.getText().length()>0) {
+                    if(!Utils.isActualNumber(headTxt.getText())) {
+                        Utils.showMessage("Please input number correctly");
+                        headTxt.requestFocus();
+                        return;
+                    }
+                    else {
+                        head = Float.parseFloat(headTxt.getText());
+                    }
+                }
+                else {
+                    head = -1;
+                }
+
+                // Check BMI value
+                if(bmiTxt.getText().length()>0) {
+                    if(!Utils.isActualNumber(bmiTxt.getText())) {
+                        Utils.showMessage("Please input number correctly");
+                        bmiTxt.requestFocus();
+                        return;
+                    }
+                    else {
+                        bmi = Float.parseFloat(bmiTxt.getText());
+                    }
+                }
+                else {
+                    bmi = -1;
+                }
+
+                showResults();
+            }
+        });
+
+        birthWeightTxt.addKeyListener(new KeyAdapter() {
+            @Override
```

Comment:
You are using same `KeyAdaptor` definition 6 times.
In this case, it would be better to define inner class and reuse it to make the code DRY (https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).